jsonnet/patch-rules.libsonnet

```diff
@@ -74,6 +81,37 @@ local patchedRules = [
       },
     ],
   },
+  {
+    name: 'thanos-query.rules',
+    rules: [
+      {
+        alert: 'ThanosQueryHttpRequestQueryErrorRateHigh',
+        'for': '1h',
```

Comment:
out of curiosity, why `1h`?

Reply:
this is based on the latest discussions in https://github.com/openshift/enhancements/pull/637 (though nothing is settled). I could make it 15m to start with.

Reply:
We do have telemetry data and can estimate `for` duration from there. However from my quick look, `1h` and `15m` don't have that much of a difference.

Reply:
I've checked over the last 24 hours: 5 clusters fired the alert and it resolved for all of them after 10 to 30 minutes. In most cases no other alert was firing.